docs/event-processing/idempotent-writer.md

```diff
@@ -0,0 +1,36 @@
+---
+seo:
+  title: Idempotent Writer
+  description: An Idempotent Writer produces an Event to an Event Streaming Platform exactly once.
+---
+
+# Idempotent Writer
+A writer produces [Events](../event/event.md) that are written into an [Event Stream](../event-stream/event-stream.md), and under stable conditions, each Event is recorded only once.
```

Comment:
Please lowercase any occurrence of "event" unless they are linked.

Reply:
@miguno IIUC this is in contrast to how Ben has been copy editing all the patterns.

Reply:
Then this should be changed in the copy-editing. :)